### PR TITLE
CHEAP-1 File singlefile created

### PR DESCRIPTION
**Pull request by mpaz-redstage**

### CODE REVIEW

File: singlefile.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+test
```
Comment (mpaz-redstage) [inline singlefile.txt]: test

File: singlefile.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+
+test
```
Comment (mpaz-redstage) [inline singlefile.txt:3]: Suggestion: Inserter a new blank line

Comment (mpaz-redstage) [reply singlefile.txt:3]: asdfasdf

Comment (mauriciopazpp) [reply singlefile.txt:3]: teste

Comment (mauriciopazpp) [reply singlefile.txt:3]: teste

Comment (mauriciopazpp) [reply singlefile.txt:3]: d

Comment (mauriciopazpp) [reply singlefile.txt:3]: d

Comment (mauriciopazpp) [reply singlefile.txt:3]: teste

Comment (mauriciopazpp) [reply singlefile.txt:3]: f

Comment (mauriciopazpp) [reply singlefile.txt:3]: f

Comment (mauriciopazpp) [reply singlefile.txt:3]: d

Comment (mauriciopazpp) [reply singlefile.txt:3]: teste

Comment (mauriciopazpp) [reply singlefile.txt:3]: teste

Comment (mauriciopazpp) [reply singlefile.txt:3]: teste

Comment (mauriciopazpp) [reply singlefile.txt:3]: d

Comment (mauriciopazpp) [reply singlefile.txt:3]: teste

**Review by mauriciopazpp — APPROVED** [review]

**Review by mauriciopazpp — CHANGES_REQUESTED** [review]
teste!

**Pull request closed without merging**